dev/oe.R

```diff
@@ -0,0 +1,150 @@
+# OE
+
+library(dplyr)
+library(labelled)
```

Comment:
Could you use `metatools` rather than labelled package please.  I'm trying to purge labelled from our scripts.  #87